torchsynth/filter.py

```diff
@@ -0,0 +1,252 @@
+"""
+Time-varying filters.
+"""
```

Comment:
What do you guys think about having this in a separate module like this? I was getting a weird sphinx error when trying to generate docs for this that wasn't occurring when it was in module, so that is weird ... need to look into that a bit more

Reply:
It's fine by me

Reply:
+1